hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/report/model/TenantUsage.java

```diff
@@ -19,6 +23,7 @@
     private long artifacts;
     private long actions;
     private long overallArtifactVolumeInBytes;
+    private Map<String, String> usageData;
```

Comment:
When you add an instance variable, please adapt `equals()` and `hashCode()` as well